server/sonar-qa-util/build.gradle

```diff
@@ -0,0 +1,28 @@
+sonarqube {
```

Comment:
I know that this is the current behavior in Maven, but this subproject should be simply excluded from analysis.

Reply:
Done